lib/metalsmith.js

```diff
@@ -134,16 +134,14 @@ module.exports = metalsmith
     }
 
     await Promise.all([
-      // GOV.UK Frontend `/assets/images` use new crown
-      copyAssets('{fonts/*,images/*,manifest.json}', {
+      copyAssets('{fonts,images}/*', {
         cwd: join(dirname(require.resolve('govuk-frontend')), 'assets'),
         dest: 'assets'
       }),
 
-      // GOV.UK Frontend `/assets/v5.0.0/images` use old crown via `src/assets`
-      copyAssets('{fonts/*,manifest.json}', {
+      copyAssets('manifest.json', {
         cwd: join(dirname(require.resolve('govuk-frontend')), 'assets'),
-        dest: 'assets/v5.0.0'
+        dest: 'assets'
       }),
```

Comment:
Appreciate this is a straight reversion to the way it was beforehand, but if we're in the mood to sneak in an optimisation, could we keep the combined glob (`{fonts/*,images/*,manifest.json}`) and reduce this to a single instance of `copyAssets`?

Reply:
Snap, if we revert https://github.com/alphagov/govuk-design-system/commit/b4c84403c0cabd46edf5fad55b39362c007efcda it'll go back to https://github.com/alphagov/govuk-design-system/pull/3525/commits/0a102d5f1b31b6a5a9cf0443f29a140002e1ffc6 💪